spec/features/bulk_updates/bulk_updates_science_keywords_spec.rb

```diff
@@ -95,7 +95,7 @@
         end
 
         it 'no longer has the removed keyword' do
-          within '.science-keywords-preview' do
+          within '#metadata-preview' do
             expect(page).to have_no_content('ATMOSPHERE ATMOSPHERIC TEMPERATURE SURFACE TEMPERATURE', normalize_ws: true)
```

Comment:
this should be separated to verify it does NOT have  ATMOPSHERE, ATMOSPHERIC TEMPERATURE, SURFACE TEMPERATURE (3 expects)

Reply:
Actually you can probably just say, `expect(page).to have_no_content('SURFACE TEMPERATURE', normalize_ws: true)`